### ISSUE TITLE
Videos do not play correctly.

### ISSUE THREAD

**Issue by viasux** [ajbura/cinny#369]
#### Describe the bug
Videos don't play, however, you can still download and open them in a new tab just fine.

#### To Reproduce
1. Click on a video
2. See an infinite loading animation

#### Expected behavior
I expect to see the video play.

#### Screenshots
![image](https://user-images.githubusercontent.com/47131867/157570202-7a0dc528-ce4c-4303-9eaf-6fc079143932.png)

#### Desktop (please complete the following information):
 - OS: [e.g. Windows, MacOS] Windows
 - Browser: [e.g. chrome, firefox] LibreWolf
 - Version: [e.g. 3.22] 1.8.0
 - Matrix homeserver: [e.g. matrix.org] yiff.social

**Comment by kfiven** [comment]
Only `.mp4` file type seems to work.

Not working: `mov`, `3gp`, `flv`, `mkv`

**Comment by ajbura** [comment]
Fixed in https://github.com/cinnyapp/cinny/commit/a417980a81c1b0ec97a0891e11b5b893c9dcce55

Although only following mimetype are played:
```
  'video/mp4',
  'video/webm',
  'video/ogg',
  'video/quicktime',
```

**Comment by viasux** [comment]
> Fixed in [a417980](https://github.com/cinnyapp/cinny/commit/a417980a81c1b0ec97a0891e11b5b893c9dcce55)
> 
> Although only following mimetype are played:
> 
> ```
>   'video/mp4',
>   'video/webm',
>   'video/ogg',
>   'video/quicktime',
> ```

I believe the file in question was an mp4 file, however I don't think ive had the issue since a recent update. I'll keep this updated if i get it again.
